Sources/Testing/ExitTests/ExitTest.swift

```diff
@@ -37,6 +37,32 @@ public struct ExitTest: Sendable {
   /// terminate the process in a way that causes the corresponding expectation
   /// to fail.
   public func callAsFunction() async -> Never {
+#if SWT_TARGET_OS_APPLE && !SWT_NO_MACH_PORTS
```

Comment:
Maybe extract all this out into a helper function?

Reply:
Sure!